tests/src/JIT/jit64/opt/cse/hugeexpr1.csproj

```diff
@@ -30,6 +30,9 @@
     <DebugType>Full</DebugType>
     <Optimize>True</Optimize>
   </PropertyGroup>
+  <PropertyGroup>
+    <JitOptimizationSensitive>true</JitOptimizationSensitive>
```

Comment:
> Do you still think it's worth changing?

Absolutely. We should not be a build targeted at architecture X on architecture Y; we should instead be cross-compiling as appropriate.